eradiate/scenes/atmosphere/onedim_solver_app.rst

```diff
@@ -0,0 +1,173 @@
+.. _sec-user_guide-onedim_solver_app:
```

Comment:
Is this file at its correct location?

Reply:
Probably not, I'll just delete it, I have no idea where that comes from.